readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -707,16 +707,21 @@ def test_init_api_project(self):
         self.assertEqual(api_project.features, [])
         self.assertFalse(api_project.ad_free)
         self.assertTrue(api_project.show_advertising)
-        self.assertEqual(api_project.environment_variables, {})
+        self.assertEqual(api_project.environment_variables(public_only=False), {})
```

Comment:
Would be good to add a test case for `public_only=True` as well.

Reply:
done!